website/docs/packages/overview.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+Reusability is an integral part of application development, offering various advantages like speeding up the development process, centralizing business logic, and introducing standardization into applications.
+
+Appsmith enhances reusability by providing a feature set, allowing you to reuse widgets, datasource, queries, JS logic, and more across your applications.
```

Comment:
What do you mean by allowing to reuse datasource? What is the feature that you have in mind for that?